en/docs/deploy/configure-an-sp-and-idp-using-configuration-files.md

```diff
@@ -429,48 +429,40 @@ service provider in the ` service provider IS ` via a file.
 
 ## Run the travelocity application
 
-Follow the instructions given below to run the travelocity application.
+{! fragments/travelocity-deploy.md !}
 
-1.  [Deploy the travelocity](TBD:../../learn/deploying-the-sample-app/#download-the-sample)
-    application. No need to register the service provider for
-    travelocity as we created the service provider via a file.
-
-2.  When you access the following link to the travelocity application,
-    you are directed to the identity provider for authentication: `
-    http://wso2is.local:8080/travelocity.com/index.jsp `
-
-    !!! note
-        Check whether you have enabled following configurations while adding the service provider 
-        in the `identity provider IS`. You can check this by navigating to **Main** -> **Identity** -> 
-        **Service Providers** -> **List** -> Click **Edit** in the corresponding SP -> 
-        **Inbound Authentication Configuration** -> **SAML2 Web SSO Configuration** -> **Edit**.
-        
-        ![SAML2 Web SSO Configuration](../../../assets/img/deploy/sp_configs.png)
-        
-        If you have `not enabled` the configs, add the following configs to travelocity as well.
-        
-        1.  Stop tomcat if it is already running.
-        2.  Navigate to `<TOMCAT_HOME>/webapps/travelocity.com/WEB­INF/classes/travelocity.properties` file 
-        and change the following configs to `false` if they are configured to `true`.
-        
-        ```
-        #Specify if SAMLResponse element is signed
-        SAML2.EnableResponseSigning=false
-        
-        #Specify if SAMLAssertion element is signed
-        SAML2.EnableAssertionSigning=false
-        
-        #Specify if AuthnRequests and LogoutRequests should be signed
-        SAML2.EnableRequestSigning=false
-        ```
+!!! note
+    Check whether you have enabled following configurations while adding the service provider 
+    in the `identity provider IS`. You can check this by navigating to **Main** -> **Identity** -> 
```

Comment:
In the majority of the docs we have used `>` instead of `->`. Therefore better to have it in a consistent manner